sass/blocks/social/_social.scss

```diff
@@ -72,4 +14,23 @@
     overflow: hidden;
     clip: rect(0 0 0 0);
   }
+
+  &__svg {
```

Comment:
again don't agree with this approach apply classes to svg. Definitely could be done differently and with less code and more maintainable. what if you need to change the image in the future? or move it to some sprite or use some other technic? you will have to rebuild it.